packages/gatsby-transformer-remark/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-transformer-remark",
   "description": "Gatsby transformer plugin for Markdown using the Remark library and ecosystem",
-  "version": "2.1.1-beta.4",
+  "version": "2.1.1-beta.5",
```

Comment:
Please revert this version bump - this is done automatically when we release new version of package